### ISSUE TITLE
No errors displayed for verilog, flycheck-compile works, the path is the same

### ISSUE THREAD

**Issue by fabiodl** [flycheck/flycheck#1812]
Hello, 
I have checked issue #1250 but it seems not to be case.

Output of flycheck-verify-setup

`Syntax checkers for buffer core.v in verilog-mode:

  verilog-verilator
    - may enable: yes
    - executable: Found at /usr/local/bin/verilator

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled
checkers.

--------------------

Flycheck version: 31
Emacs version:    26.3
System:           x86_64-pc-linux-gnu
Window system:    x
`

Verilator version:
4.036 2020-06-06 rev v4.036-7-g369ce6af

The status bar shows no errors (FlyC) when the file clearly has. When editing the file it temporarily shows activity (FlyC*). 

example of output from flycheck-make
```
*- mode: compilation; default-directory: "~/git/verilog/selgaro/" -*-
Compilation started at Tue Jul 21 13:02:14

verilator --lint-only -Wall /home/fabio/git/verilog/selgaro/core.v
%Error: /home/fabio/git/verilog/selgaro/core.v:20:23: syntax error, unexpected '/', expecting ';'
   20 |    output reg[20:0] ia/2;
      |                       ^
%Error: Exiting due to 1 error(s)

Compilation exited abnormally with code 1 at Tue Jul 21 13:02:14
```

I tried 
(setq flycheck-find-checker-executable "/usr/local/bin/verilator_bin")
but it made no difference.
I have a single verilator installation (this one in /usr/local). 

The error persists by running emacs -q and then evaluating
```
(require 'package)
(package-initialize)
(global-flycheck-mode)
```

**Comment by toshipp** [comment]
I think #1691 resolves this.
